### DISCUSSION TITLE
Can this be compiled for Android and work?

### DISCUSSION THREAD

**Discussion by therealrobster** [Orama-Interactive/Pixelorama#1022] [Q&A] [answered]
Hi all, 

I have a Samsung tablet with an S Pen.  Which is just a wacom rebranded. 

I'm wondering, had anyone successfully compiled and run Pixelorama on Android?

Thank you

**Comment by OverloadedOrama** [comment]
Hello!
It can indeed be compiled for Android, I have successfully done it many times for my phone, and I would love to official support Android in the future. However, the current state of the UI is not very touch screen friendly, and not suitable for devices with small screens like phones. It should work fine on a tablet with a pen though, but there are still some issues that we need to fix, like supporting touch gestures on the canvas to move, pan and perhaps even rotate.

**Reply by therealrobster** [reply]
That's amazing. Thank you. Well,  I have a very large display,  and a pen. So I'd love to try. 

Are there instructions on how to build for Android, so I can give it a try? 

Thanks again!

**Reply by OverloadedOrama** [reply, marked as answer]
You need to download the latest version of Godot stable (4.2.2 at the time of writing this), and then you can follow the instruction on their documentation: https://docs.godotengine.org/en/stable/tutorials/export/exporting_for_android.html

**Reply by therealrobster** [reply]
Thank you. I'll give it a try today. I read your previous comment again. Definitely things like palm rejection, two finger zoom, two finger rotation and that kind of thing would be required to make it really usable. But for now I'll give it a try and provide some feedback if you'd like.

**Reply by OverloadedOrama** [reply]
Of course, feedback would be very helpful!

**Comment by toihr** [comment]
Just wanted to add here the steps outlined by @OverloadedOrama still work well, from a 3rd Party Perspective. Suprisingly even resizing the canvas by pulling and pusshing 2 fingers appart/together seems to work. 

I would perhaps be looking at pen functionality in Godot idk if that is even in a Godot API.
